### PR TITLE
Define behavior when rotating camera.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fixes https://github.com/w3c/mediacapture-main/issues/660.

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/mediacapture-main/pull/760.html" title="Last updated on Dec 17, 2020, 4:03 PM UTC (6c12505)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-main/760/826b76a...jan-ivar:6c12505.html" title="Last updated on Dec 17, 2020, 4:03 PM UTC (6c12505)">Diff</a>

### CODE REVIEW

File: getusermedia.html [hunk 1]
```diff
@@ -1736,6 +1736,22 @@ <h2>Constrainable Properties</h2>
             </tr>
           </tbody>
         </table>
+        <p>On systems where it's desirable to sometimes automatically flip the X
+        and Y axis of the resulting captured video in response to ongoing
+        environmental factors, the {{width}}, {{height}} and {{aspectRatio}}
+        constraints and capabilities MUST remain unaffected in all algorithms
+        and be considered in the <a>primary orientation</a> only, except for the
+        {{MediaStreamTrack/getSettings()}} algorithm where settings for these
+        constrainable properties MUST be flipped, if necessary to match the
```
Comment (alvestrand) [inline getusermedia.html:1745]: this comma shouldn't be there.

Comment (jan-ivar) [inline getusermedia.html:1745]:
```suggestion
        constrainable properties MUST be flipped if necessary to match the
```

File: getusermedia.html [hunk 2]
```diff
@@ -1736,6 +1736,22 @@ <h2>Constrainable Properties</h2>
...
+        returned dimensions of the captured video at any point in time.</p>
+        <p>The <dfn>primary orientation</dfn> of a system that supports flipping
+        the X and Y axis of resulting captured video is defined by the user
+        agent for the particular system.</p>
+        <div class="note">
+          <p>On systems that support automatic switching between landscape and
+          portrait mode, user agents are encouraged to make landscape mode the
```
Comment (youennf) [inline getusermedia.html:1752]:
```suggestion
          portrait mode, User Agents are encouraged to make landscape mode the
```

File: getusermedia.html [hunk 3]
```diff
@@ -1736,6 +1736,22 @@ <h2>Constrainable Properties</h2>
...
+        <p>On systems where it's desirable to sometimes automatically flip the X
+        and Y axis of the resulting captured video in response to ongoing
+        environmental factors, the {{width}}, {{height}} and {{aspectRatio}}
+        constraints and capabilities MUST remain unaffected in all algorithms
+        and be considered in the <a>primary orientation</a> only, except for the
+        {{MediaStreamTrack/getSettings()}} algorithm where settings for these
+        constrainable properties MUST be flipped, if necessary to match the
+        returned dimensions of the captured video at any point in time.</p>
+        <p>The <dfn>primary orientation</dfn> of a system that supports flipping
+        the X and Y axis of resulting captured video is defined by the user
```
Comment (youennf) [inline getusermedia.html:1748]:
```suggestion
        the X and Y axis of resulting captured video is defined by the User
```

File: getusermedia.html [hunk 4]
```diff
@@ -1736,6 +1736,22 @@ <h2>Constrainable Properties</h2>
...
+        agent for the particular system.</p>
```
Comment (youennf) [inline getusermedia.html:1749]:
```suggestion
        Agent for the particular system.</p>
```

**Comment by henbos** [comment]
Please file follow-up about orientation bla bla bla

**Pull request merged**